anvil/src/udev.rs

```diff
@@ -1034,7 +1044,8 @@ impl AnvilState<UdevData> {
                 warn!("Error during rendering: {:?}", err);
                 match err {
                     SwapBuffersError::AlreadySwapped => true,
-                    SwapBuffersError::TemporaryFailure(err) => matches!(
+                    // If the device has been deactivated or a vt switch is in progress do not reschedule
+                    SwapBuffersError::TemporaryFailure(err) => !matches!(
```

Comment:
I think this is too simplistic. E.g. after resuming the session my nvidia gpu will cause TemporaryFailures for one or two frames until operation continues as normal.

So I think we unfortunately need to differentiate here between TemporaryErrors, when the device should be active or should not be active.

Reply:
Ok, makes sense. If we only disable re-schedule for not active it should be fine, though it will render a few frames that will never make it to the screen. But that is not really an issue I guess.

Reply:
done